src/test/frontend/replicasetdetail/replicaset_service_test.js

```diff
@@ -19,52 +19,16 @@ describe('Replica set service', () => {
   let service;
   /** @type {!md.$dialog} */
   let mdDialog;
-  /** @type {!angular.$q} */
-  let q;
-  /** @type {!angular.$httpBackend} */
-  let httpBackend;
 
   beforeEach(() => {
     angular.mock.module(replicaSetDetailModule.name);
 
-    angular.mock.inject((kdReplicaSetService, $mdDialog, $q, $httpBackend) => {
+    angular.mock.inject((kdReplicaSetService, $mdDialog) => {
       service = kdReplicaSetService;
       mdDialog = $mdDialog;
-      q = $q;
-      httpBackend = $httpBackend;
     });
   });
 
-  it('should show delete dialog and delete object', (doneFn) => {
```

Comment:
@bryk, @floreks Before you merge: guys, I have some trouble with those tests. They both fail on my machine, even though they are unrelated to this pull request. What I do not get:
* to my understanding a mock-promise is attached to the `showDeleteDialog()` which is O.K. ... But why is there an expected 'DELETE' request for the mock httpBackend? There is no call which would actually delete a `Replicaset`, it's just a mocked dialog with an attached mocked promise response ... Am I missing something?

httpBackend.flush() fails on my machine, because there's no request.